.github/scripts/pin_github_actions.py

```
@@ -0,0 +1,277 @@
+            # Process steps
+            for step in job.get('steps', []):
+                if 'uses' in step:
+                    original_action = step['uses']
+                    pinned_action = pin_action(original_action)
+                    if original_action != pinned_action:
+                        # Replace in original content to preserve formatting
+                        content = content.replace(f"uses: {original_action}", f"uses: {pinned_action}")
+                        modified = True
+
+    # Process reusable workflows
+    for job_name, job in workflow.get('jobs', {}).items():
```

Ruff (0.13.1)

Loop control variable `job_name` not used within loop body

Rename unused `job_name` to `_job_name`

(B007)